app/main.py

```diff
@@ -2,0 +1,12 @@
+def copy_file(command: str) -> None:
+    command_ls = command.split()
+    if (
+            len(command_ls) == 3
+            and command_ls[0] == "cp"
+            and command_ls[1] != command_ls[2]
+    ):
+        with (
+            open(command_ls[1], "r") as r_file,
+            open(command_ls[2], "w") as w_file
+        ):
+            w_file.write(r_file.read())
```

Comment:
The code does not handle the case where the source file does not exist. This will cause a `FileNotFoundError`. You should add a check to see if the source file exists before trying to open it.